GUIDELINES_AND_INFO.md

```diff
@@ -8,4 +8,13 @@ This is
 - an API for Scratch account verification
 
 ## Guidelines
-None yet.
+(NOT DONE)
+Please read our [Code of Conduct](https://github.com/ScratchVerifier/ScratchVerifier/blob/develop/CODE_OF_CONDUCT.md) before proceeding. 
+The following behaviour is not encouraged:
+- Posting issues with very simple descriptions, or unconstructive criticism. We would like you to in detail explain what can be done, what is wrong and what could get an update.
+- Irrelevant discussions: Please move to our Gitter channel off-topic (coming soon) to continue any discussions. Any long off-topic discussions could make the discussion get marked as offtopic, removed or the issue locked outright if the problem is worse enough
```

Comment:
We have a Discord server, why are we also doing Gitter?

Reply:
I did not see a need for it for now, but if we could make it better then sure

Reply:
fixed